### PR TITLE
Added typescript webhook example

### PR DESCRIPTION
**Pull request by paulasjes-stripe**
r? @rattrayalex-stripe

### CODE REVIEW

File: examples/webhook-signing/express-ts.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+const app: express.Application = express();
+
+// Only use the raw body parser for webhooks
+app.use(
+  (
+    req: express.Request,
+    res: express.Response,
+    next: express.NextFunction
+  ): void => {
+    if (req.originalUrl === '/webhooks') {
```
Comment (rattrayalex-stripe) [inline examples/webhook-signing/express-ts.ts]: Is this our best-practice? Does line 55 not work here?

File: examples/webhook-signing/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "target": "es5" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */,
+    "module": "commonjs" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
+    "types": ["../../types/2019-12-03"],
+
+    /* Strict Type-Checking Options */
+    "strict": true /* Enable all strict type-checking options. */,
+    "esModuleInterop": true /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */,
```
Comment (rattrayalex-stripe) [inline examples/webhook-signing/tsconfig.json]: I don't think this is needed any longer

File: examples/webhook-signing/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "target": "es5" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */,
+    "module": "commonjs" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
+    "types": ["../../types/2019-12-03"],
```
Comment (rattrayalex-stripe) [inline examples/webhook-signing/tsconfig.json]: This shouldn't be needed once it lands, but I guess we can keep it in till then?

File: examples/webhook-signing/express-ts.ts (hunk 1 above)
Comment (paulasjes-stripe) [reply examples/webhook-signing/express-ts.ts]: Line 55 does work, but I've seen people get tripped up by how express' middleware works. Having this block here will hopefully nip that confusion in the bud (and makes it so all other routes use the json body parser).

File: examples/webhook-signing/tsconfig.json (hunk 3 above)
Comment (paulasjes-stripe) [reply examples/webhook-signing/tsconfig.json]: Yeah I wasn't sure if this is necessary or not, happy to remove once v8 lands.

File: examples/webhook-signing/tsconfig.json (hunk 2 above)
Comment (paulasjes-stripe) [reply examples/webhook-signing/tsconfig.json]: It's needed for line 5 of `express-ts.ts` where we import `dotenv`.

File: examples/webhook-signing/express-ts.ts [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    }
+  }
+);
+
+// Stripe requires the raw body to construct the event
+app.post(
+  '/webhooks',
+  bodyParser.raw({type: 'application/json'}),
+  (req: express.Request, res: express.Response): express.Response | void => {
+    const sig: string = req.headers['stripe-signature'] as string;
```
Comment (rattrayalex-stripe) [inline examples/webhook-signing/express-ts.ts]: this is fixed; you no longer need to cast!

**Pull request closed without merging**